source/Planet.cpp

```diff
@@ -122,8 +132,37 @@ void Planet::Load(const DataNode &node)
 				continue;
 		}
 		
+		if(key == "port")
+		{
+			port.name = hasValue ? value : "Space_port";
+			for(const auto &grand : child)
+			{
+				if(grand.Token(0) == "shields")
+					port.recharge = static_cast<Port::RechargeType>(port.recharge | Port::Shields);
```

Comment:
Can you not just do `port.recharge |= Port::Shields;`?

Reply:
Nope cause `port.recharge` is a enum. If I make it an `int` it will work, but then we're losing a bit of type safety which is why I didn't do it initially.

Reply:
Well seeing as how a lot of other enums in the game (e.g. ShipEvent) are actually just ints that we use bitwise functions on, I'd definitely prefer if the same could be done here.